### PR TITLE
Added listener to follow after a reply

### PR DESCRIPTION
**Pull request by petermein**
Added listener to add setting to a user

### CODE REVIEW

**Comment by petermein** [comment]
Referencing: https://github.com/flarum/english/pull/57

File: js/forum/src/main.js [hunk 1]
```diff
@@ -17,8 +19,10 @@ app.initializers.add('subscriptions', function() {
   addSubscriptionBadge();
   addSubscriptionControls();
   addSubscriptionFilter();
+  console.log('subscription');
```
Comment (franzliedke) [inline js/forum/src/main.js]: uh oh ;)

**Comment by petermein** [comment]
As mentioned in the https://github.com/flarum/core/issues/310 i updated this to include the changes. But currently i added a toggle button for this setting. I don't know what the best option is to toggle this. Or how to fix the ordering. When i use the example code of @tobscure the buttons are nested inside the div of the Notification grid. Which is not ideal.

**Comment by tobyzerner** [comment]
Thanks @petermein! I fixed it up and squash-merged it in. flarum/flarum-ext-subscriptions@98276f9

**Pull request closed without merging**